pkg/sql/parser/sql.y

```diff
@@ -7514,6 +7516,25 @@ multiple_set_clause:
     $$.val = &tree.UpdateExpr{Tuple: true, Names: $2.nameList(), Expr: $5.expr()}
   }
 
+// %Help: REASSIGN OWNED - change ownership of all objects
+// %Category: Priv
```

Comment:
Is this the correct category? `GRANT` is under Priv but `ALTER` is under DDL.

Reply:
This is fine to me!